### DISCUSSION TITLE
Archival of old code

### DISCUSSION THREAD

**Discussion by Scarjit** [united-manufacturing-hub/united-manufacturing-hub#686] [General]
While progressing with our stack, we sometimes deprecate code.

I think we should archive this code in a separate repository (so it doesn't bloat our main one).
This would enable other users to re-use our open-source code and also keep the contributions of all developers.

**Comment by Scarjit** [comment]
@JeremyTheocharis @kruegeralex

**Reply by JeremyTheocharis** [reply]
agreed. I suggest to move python-sdk out of this repository. is there anything else?

**Reply by Scarjit** [reply]
At the moment just python-sdk

**Reply by JeremyTheocharis** [reply]
then let's go for it. can you create your own repos?

**Reply by Scarjit** [reply]
Yes, i can
